src/material/core/common-behaviors/common-module.ts

```diff
@@ -65,7 +66,8 @@ export class MatCommonModule {
   constructor(
       highContrastModeDetector: HighContrastModeDetector,
       @Optional() @Inject(MATERIAL_SANITY_CHECKS) sanityChecks: any,
-      @Inject(DOCUMENT) document: any) {
+      @Inject(DOCUMENT) document: any,
+      private _platform?: Platform) {
```

Comment:
Are we planning on making this required in the future? or do you think it's always okay being optional?

Reply:
Yeah, it should be required in the future, I think I just missed it before. Adding it now.